ledger/controllers/surveyor.js

```diff
@@ -422,7 +427,8 @@ const create = async (debug, runtime, surveyorType, payload, parentId) => {
       surveyorType: surveyorType,
       active: surveyorType !== 'contribution',
       available: true,
-      payload: payload
+      payload: payload,
+      frozen: false
```

Comment:
I think we can remove this and everything having to do with frozen from `ledger`, only `eyeshade` needs to be aware of frozen state